Elevate Query Builder to GA (#8427)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## January 2026

> - **New:** The [Query Builder](/docs/explore/navigate-dbt-insights#query-builder) in dbt <Constant name="query_page" /> is now generally available (GA). With Query Builder, you can build SQL queries against the Semantic Layer without writing SQL code.

Reviewer: ```suggestion
- **New:** The [query builder](/docs/explore/navigate-dbt-insights#build-a-query) in dbt <Constant name="query_page" /> is now generally available (GA), enabling you to build SQL queries against the Semantic Layer without writing SQL code.
```
